apps/prairielearn/src/sync/fromDisk/assessments.ts

```diff
@@ -160,9 +147,9 @@ function getParamsForAssessment(
             maxAutoPoints: question.maxAutoPoints ?? null,
             autoPoints: question.autoPoints ?? null,
             manualPoints: question.manualPoints ?? null,
-            forceMaxPoints: question.forceMaxPoints || false,
-            triesPerVariant: question.triesPerVariant || 1,
-            advanceScorePerc: question.advanceScorePerc,
+            forceMaxPoints: question.forceMaxPoints ?? false,
+            triesPerVariant: question.triesPerVariant ?? 1,
```

Comment:
this `triesPerVariant` change is technically breaking, do we care?

Reply:
How so, because of how `0` is handled?

Reply:
Yeah.

Reply:
I checked our production data with the following query:

```sql
SELECT * FROM assessment_questions WHERE tries_per_variant <= 0;
```

It returned zero rows. I'm inclined to say we should add additional validation (maybe in the JSON schema, maybe also in the database schema) to require this to be >= 1; that can/should be done in a separate PR.